includes/class-admin-settings.php

```diff
@@ -44,11 +44,14 @@ class Admin_Settings {
 	 * The Constructor.
 	 */
 	public function __construct() {
-		add_action( 'admin_menu', array( $this, 'register_admin_menu' ) );
+		add_action( is_multisite() ? 'network_admin_menu' : 'admin_menu', array( $this, 'register_admin_menu' ) );
 		add_action( 'admin_init', array( $this, 'reset_settings' ) );
 		add_action( 'admin_init', array( $this, 'register_settings' ) );
 		add_action( 'admin_notices', array( $this, 'reset_admin_notice' ) );
 		add_action( 'admin_enqueue_scripts', array( $this, 'admin_enqueue_scripts' ) );
+
+		add_action( 'admin_init', array( $this, 'update_settings' ) );
+		add_action( 'network_admin_edit_aspireupdate-settings', array( $this, 'update_settings' ) );
```

Comment:
Double-checking if this is necessary for this issue or if this fixes a different issue?

Reply:
This is necessary for saving from multisite and provides a consistent manner of saving for both single and multisite.